spec/index.bs

```diff
@@ -82,9 +82,39 @@ Issue: figure out if we need this section here. CSP has none, SRI has one and it
     libraries that perform auxiliary tasks (e.g. analytics, performance
     monitoring). To ensure that none of these components introduces DOM
     XSS vulnerabilities, author defines a Trusted Type policy in the
-    templating policy and enables the enforcement for the DOM sinks.
-*   Issue: Something about reducing the review surface
-
+    templating policy and enables enforcement for the DOM sinks.
+*   A large team maintains a complex client-side application.
+    They create a number of Trusted Types policies so that
+    *   A strict auto-escaping template system can
+        *   produces {{TrustedHTML}},
+        *   exempts {{TrustedHTML}} from extra escaping,
+        *   checks that values for HTML attributes like
+            {{HTMLHyperlinkElementUtils/href|href}} and
+            {{HTMLImageElement/src|HTMLImageElement.src}}
+            are {{TrustedURL}} or {{TrustedScriptURL}} or are
+            convertible to the same via the [=default policy=].
+    *   A carefully reviewed HTML sanitizer can
+        *   produce {{TrustedHTML}} from untrusted strings
+        *   exempt {{TrustedHTML}} values from extra sanitizing.
+    *   Thorny legacy code that has a long history of running in
+        production without reported vulnerabilities can continue to
+        run with enforcement enabled.
+
+    The team consolidates the policy implementations and the
+    safe abstractions that use them in a small number of heavily
+    reviewed files and requires extra approval for commits that
+    affect these files.
+
+    The need to create trusted values to affect [=injection sinks=],
+    combined with additional scrutiny on changes that affect policy
+    code, incents developers to find novel ways to use safe
```

Comment:
incents developers to use safe abstractions (consider removing "to find novel ways", that to me sounds like finding creative abuse methods, or bypasses).